.github/workflows/scheduled-container-scanning.yml

```diff
@@ -0,0 +1,54 @@
+#
+# SPDX-FileCopyrightText: 2023 Lifely
+# SPDX-License-Identifier: EUPL-1.2+
+#
+name: Scheduled container scanner
+
+on:
+  schedule:
+    # Runs at 9:00 PM UTC from Monday to Friday
+    - cron: "0 21 * * 1-5"
+  workflow_dispatch:
+
+env:
+  JAVA_VERSION: '17'
+  CONTAINER_REGISTRY_URL: 'ghcr.io/infonl'
+  APPLICATION_NAME: 'zaakafhandelcomponent'
+
+permissions:
+  # Required for uploading SARIF reports
+  security-events: write
+
+jobs:
+  build:
+    name: Run Trivy code scanner
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Docker Login
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.CONTAINER_REGISTRY_URL }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Restore Docker Image
+        uses: actions/cache/restore@v3
+        with:
+          path: docker-image.tar
+          key: docker-image-${{ github.repository }}-${{ github.ref_name }}-${{ github.run_number }}
```

Comment:
Dit gaat niet werken? Het huidige GitHub run nr zijn niet dezelfde als die vh Docker image wat je hier wil scannen. Sterker nog, er is helemaal geen Docker image gebouwd in deze run. Er zit ook niks in de cache.

Wat je wil is de laatste releasde Docker image van ons uit onze eigen repo pullen. Dat kan want we gebruiken de 'latest' tag.